app/src/main/java/com/ivik/passwordmanager/MainActivity.java

```diff
@@ -22,18 +27,79 @@
     private SharedPreferences.Editor editor;
     private PasswordManager passwordManager;
     private String userKey;
+    private List<Account> accounts;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        askForNewPassword();
         if (SharedPreferencesHelper.isFirstTimeLaunch(this)) {
             askForNewPassword();
         }
-//        else {
-//            askForPassword();
-//        }
+        else {
+            askForPassword();
+        }
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    private void addNewAccount() {
+        final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle("Add a new account:");
+
+        LinearLayout linearLayout = new LinearLayout(this);
```

Comment:
Dialogy tiez vedia mat .xml layouty, cize nemusis ho tu programaticky vytvarat. Je to tak lepsie. Pozri [tu](https://developer.android.com/guide/topics/ui/dialogs#CustomLayout).

Reply:
Skusal som to, ale nevedel som tie EditText-y najst cez findViewById.

Reply:
Vyskusam znova.